SharedCode/src/commonMain/kotlin/ApplicationContract.kt

```diff
@@ -8,6 +8,7 @@ interface ApplicationContract {
         fun setDepartureDropdown(stationList: List<String>)
         fun setArrivalDropdown(stationList: List<String>)
         fun populateDeparturesTable(departuresList: List<DepartureInformation>)
+        fun createAlertMessage(alertMessage: String)
```

Comment:
Picky but I'd probably name this showAlertMessage as it's a slightly better description of what it does, do you agree?

Reply:
Also, you should create a stub of this function in ViewController.swift so that the iOS app can still build successfully 🙂